app/js/OutputCacheManager.js

```diff
@@ -281,7 +282,16 @@ module.exports = OutputCacheManager = {
           contentDir,
           outputFilePath,
           pdfSize,
+          timings.compile,
           function (err, result) {
+            if (err && err instanceof TimedOutError) {
+              logger.warn(
+                { err, outputDir, stats, timings },
```

Comment:
It's good that we have the outputDir, I was going to say that about the previous PR - that it would be good to get the project id so we can have a look at some of these and see what is going on.